pubsub/api/Pubsub.Samples/QuickStart.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START pubsub_quickstart_create_topic]
+
+using Google.Cloud.PubSub.V1;
+using System;
+
+public class QuickStartSample
```

Comment:
This one is very similar to the CreateTopic sample, I think we can keep one with both region tags in it.
You can keep the code from this one which prints some info to the console, but the file and class names from the other one are better.